src/pages/documentation/mobile-core/release-notes.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## April 14, 2023
+
+### Android Mobile Core 2.1.2
+
+* Fixed an issue that prevented some in-app messaging show frequency rules from working correctly.
```

Comment:
What exactly do we mean here by "Show frequency rules?"

Is it like the IAM frequency rules weren't working properly?

Reply:
These are rules which are configured to be shown once or on every launch.

Reply:
@sbenedicadb Pinging you for the correct name.

Reply:
@calebk1 that's correct.  show frequency is a section in part of the IAM publishing process where the marketeer determines how often this message will be shown.  can include values like "only once", "x number of times", "until y event happens", or simply "show it every time the conditions are met".